tests/test_datetime.py

```diff
@@ -2,17 +2,14 @@
 
 import copy
 import datetime as std_datetime
-from decimal import Decimal
 import pickle
+from decimal import Decimal
 from typing import SupportsIndex, Type
 
 import hightime
```

Comment:
Ignoring in this PR. Ordering of imports is fixed in #59.

Reply:
I think what happened here is that ni-python-styleguide didn't know the project name until I added it to `pyproject.toml`